### DISCUSSION TITLE
How to customize the labels of the X-axis, I can't find any examples using VIEW, they are all examples of Compose

### DISCUSSION THREAD

**Discussion by xiaoyvyv** [patrykandpatrick/vico#479] [Questions] [answered]
How to customize the labels of the X-axis, I can't find any examples using VIEW, they are all examples of Compose，Thanks！

**Comment by patrickmichalik** [comment, marked as answer]
Hello! Please see [here](https://github.com/patrykandpatrick/vico/blob/master/sample/src/main/java/com/patrykandpatrick/vico/sample/showcase/charts/Chart6.kt#L93).
